### ISSUE TITLE
Cannot access nested class

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#259]
The type checker complains about the `self.B()` call in the program below:

```
class A:
    class B: pass
    def f(self) -> None:
        self.B()
```

**Comment by ilevkivskyi** [comment]
Another example of this bug:
```python
class Outer:
    class Inner:
        pass

Outer().Inner  # E: Member "Inner" is not assignable
```

**Comment by ethanhs** [comment]
This is the same as #3635 fixed in #3636, unless I'm mistaken.

**Comment by ilevkivskyi** [comment]
Yes, both examples work correctly now.
